cg/constants/scout.py

```diff
@@ -146,11 +137,14 @@ class ScoutUploadKey(StrEnum):
 )
 
 NALLO_SAMPLE_TAGS: dict[str, set[str]] = dict(
-    alignment_file={"alignment_haplotags"},
+    alignment_path={AlignmentFileTag.BAM, "haplotags"},
+    assembly_alignment_path={AlignmentFileTag.BAM, "assembly"},
     d4_file={"mosdepth_d4"},
+    hificnv_coverage={"hificnv", "bigwig"},
+    paraphase_alignment_path={AlignmentFileTag.BAM, NalloAnalysisTag.PARAPHASE},
+    minor_allele_frequency_bigwig={"hificnv", "bigwig", "maf"},
```

Comment:
💯

Reply:
Aahem, sorry, if I think again the most snug would be 
`minor_allele_frequecny_wig` to parallel the `coverage_wig` etc. If this is still open and not too much to ask - it's not like it would be that bad to have `_bigwig`. 😊